traveler-ios-ui/TravelerKitUI/ViewControllers/ReceiptViewController.swift

```diff
@@ -24,7 +24,7 @@ class ReceiptViewController: UIViewController {
 
         titleLabel.text = receipt?.order.products.first?.title
         dateLabel.text = receipt.flatMap { DateFormatter.longFormatter.string(from: $0.order.createdDate) }
-        confirmationLabel.text = receipt?.order.orderNumber
+        confirmationLabel.text = receipt?.order.referenceNumber
         //emailLabel.text = receipt?.customerContact.email
```

Comment:
Remove commented out code

Reply:
Leaving it there because @ata-n might need it since he didn't outright delete it